Blizzard.IO.RabbitMQ/NetqRabbitConfigurator.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using Blizzard.IO.RabbitMQ.Entities;
+using EasyNetQ;
+using EasyNetQ.Topology;
+using Microsoft.Extensions.Logging;
+
+namespace Blizzard.IO.RabbitMQ
+{
+    public class NetqRabbitConfigurator : IRabbitConfigurator
+    {
+        private readonly IBus _netqBus;
+        private readonly ILogger _logger;
+
+        public NetqRabbitConfigurator(IBus netqBus, ILoggerFactory loggerFactory)
+        {
+            _netqBus = netqBus;
+            _logger = loggerFactory.CreateLogger(typeof(NetqRabbitConfigurator));
```

Comment:
Use the non static override of this funciton (the one that gets the string)

Reply:
f